picard/coverart/providers/caa.py

```diff
@@ -154,9 +156,11 @@ def load(self):
         self.update_caa_types()
 
     def save(self):
-        config.setting["caa_image_size"] =\
+        config.setting["caa_image_size"] = \
```

Comment:
Change unrelated to the issue, it would be better to keep those for another patch, mixing such changes make PRs harder to review. No need to revert as it is a minor change, but try to keep this in mind for future patches.